app/uk/gov/hmrc/exports/models/declaration/Transport.scala

```diff
@@ -42,4 +42,7 @@ case class Transport(
 
 object Transport {
   implicit val format: OFormat[Transport] = Json.format[Transport]
+
+  val optionNone = "option_none"
+  def typeApplicable(value: String): Boolean = value != optionNone
```

Comment:
What do you think about moving this method to the class? You could then write `transport.isApplicable`

Reply:
That would be better.  Thanks

Reply:
Actually, it would have to be something like `def isMeansOfTransportOnDepartureApplicable: Boolean = meansOfTransportOnDepartureType.exists(_ != Transport.optionNone)` as there may be other values that are not applicable.

Reply:
Created method on Transport class